src/main/rx/cyrf6936_spektrum.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#define DSM_BIND_TIMEOUT_US 10000
+#define DSM_SYNC_TIMEOUT_US 20000
+#define DSM_RECV_LONG_TIMEOUT_US 18010
+#define DSM_RECV_MID_TIMEOUT_US 7000
+#define DSM_RECV_SHORT_TIMEOUT_US 4010
+#define DSM_RECV_TIMEOUT_OFFSET_US 1000
+
+#define DSM_MAX_MISSED_PACKETS 31
+
+#define DSM_MAX_RF_CHANNEL 0x4F
+#define DSM_MAX_CHANNEL_COUNT 12
+#define DSM_INITIAL_BIND_CHANNEL 0x0D
+
+#define DSM_TELEMETRY_FRAME_RPM 0x7E
+#define DSM_TELEMETRY_TIMEOUT_US 1500
+#define DSM_TELEMETRY_TIME_US 176000
+
+typedef struct spektrumConfig_s {
+    uint8_t protocol;
```

Comment:
Sorry to come back to this over and over again, but there's a lot of ground to cover in this review.

These settings should be exposed in `src/main/interface/settings.c`. Even if users are very unlikely to ever want to set them manually, if they are exposed they will be added to the `diff`, and then can be re-applied after a firmware update. Otherwise the user will have to re-bind the RX every time they update the firmware.

Reply:
No worries, I actually added it for debugging purposes, but then removed it for the final pull request....
Updated with settings.c changes.